core/ui_mixins/quick_panel.py

```diff
@@ -500,3 +500,30 @@ def on_selection(self, commit):
     def on_selection_async(self, commit):
         sublime.active_window().run_command("hide_panel", {"panel": "output.show_commit_info"})
         self.on_done(commit)
+
+
+def show_stash_panel(entries, on_done):
```

Comment:
Do we really need a new function to show stash if `entries` are to be provided?  `show_paginated_panel` should just work. If you really want to implement a panel for stashes. Then it should be something similar to `show_branch_panel` which only takes the `on_done` argument.